doc/usage_rsmexplain.rst.inc

```diff
@@ -0,0 +1,65 @@
+.. _usage_rsmexplain:
+
+``rsmexplain`` - Explain non-linear models
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+RSMTool provides the ``rsmexplain`` command-line utility to generate explanation report for explain data using a model already trained using the ``rsmtool`` utility. This can be useful how the model decision have been impacted by each feature and weight of each feature in the final target.
+
+``rsmexplain`` The sampling explainer is model agnostic and should, in principle, work for any type of model. RSMExplain currently only supports regressors.
```

Comment:
```suggestion
``rsmexplain`` uses the sampling explainer which is model agnostic and should, in principle, work for any type of model. However, ``rsmexplain`` currently only supports regressors since they are the most popular model type used for automated scoring.
```